Dockerfile

```diff
@@ -95,7 +95,14 @@ RUN make -j 16 && make check && make install
 RUN rm -rf /tmp/gmap-gsnap /tmp/gmap-gsnap-2017-11-15.tar.gz
 RUN gsnapl --version
 
+# For srst2. Samtools line is from https://github.com/dahak-metagenomics/dahak/blob/master/docker_files/srst2/Dockerfile.
+# TODO: Update to install forked python 3 compatible srst2 repo, verify that different samtools produces same output  
+RUN curl -O -L https://sourceforge.net/projects/samtools/files/samtools/0.1.18/samtools-0.1.18.tar.bz2 && tar xvfj samtools-0.1.18.tar.bz2 && cd samtools-0.1.18 && make && cp samtools /usr/local/bin &&  cp bcftools/bcftools /usr/local/bin && cd misc/ &&  cp *.pl maq2sam-long maq2sam-short md5fa md5sum-lite wgsim /usr/local/bin/ && cd 
```

Comment:
This line is not very readable because it is too long :) Also, samtools is already being installed further up in this Dockerfile and you are overwriting that installation.
(a) It would be cleaner to just edit the first installation to use the version you need. Apparently samtools is not used anywhere other than the AMR step anyway, so it shouldn't cause problems. Even better, check if you can make SRST2 work with a newer version of samtools, as you mention in your TODO.
(b) Please break the long line up into shorter lines by using more docker-like syntax, e.g. see the use of RUN and WORKDIR in the first installation of samtools.

Reply:
Sure, addressed this in later commits.